### DISCUSSION TITLE
Nextcloud Photos [Improvement]

### DISCUSSION THREAD

**Discussion by tetebueno** [Freika/dawarich#566] [Ideas]
**Describe the bug**
Integrate Nextcloud Photos just like Immich or Photoprism.

**Version**
0.20.1

**To Reproduce**
Not a bug per-se, but an idea for an improvement. I believe there's no REST API for NC's Photos, but maybe it can be achieved useng WebDAV.

**Expected behavior**
Have an extra box to configure NC Photos' service as an alternative to Immich and Photoprism.

**Screenshots**
N/A

**Logs**
N/A

**Additional context**
N/A

**Comment by danotrampus** [comment]
Many of us use Memories for managing our photo libraries. They provide several API endpoints that could be use to achieve this goal: https://github.com/pulsejet/memories/blob/master/appinfo/routes.php#L62

**Reply by tetebueno** [reply]
Didn't want to mention that on the original issue, but yes, I also use Memories. I'd be totally in with going with Memories integration.

**Comment by mebitek** [comment]
I use memories too. should be nice to have nextcloud integration

**Comment by ubrfkr** [comment]
Are there any plans to implement this? I know that many people use Nextcloud for photo management.

**Reply by Freika** [reply]
Not in the nearest future, but I'll research Nextcloud integration at some point, yes
